src/hacbs/components/ApplicationDetails/ApplicationModal.tsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import { TextVariants, Text, Button, Modal, TextContent } from '@patternfly/react-core';
+import imageUrl from '../../imgs/application-overview.svg';
+
+export const HACBS_APPLICATION_MODAL_KEY = 'hacbs/showApplicationModal';
+
+interface ApplicationModalProps {
+  showApplicationModal: boolean;
+  onClose: () => void;
+}
+
+const ApplicationModal: React.FC<ApplicationModalProps> = ({ showApplicationModal, onClose }) => {
+  const footer = (
+    <div className="pf-u-display-flex pf-u-justify-content-center pf-u-flex-fill">
+      <Button
+        data-testid="hacbs-getting-started-modal-dismiss"
+        aria-label="get started"
+        onClick={onClose}
+      >
+        Got it, thanks
+      </Button>
+    </div>
+  );
+
+  return (
+    <Modal
+      aria-label="commits-getting-started-modal"
```

Comment:
Aria labels should be human readable text as screen readers will read this value.